policyengine_us/parameters/gov/states/dc/dhs/tanf/standard_payment/amount.yaml

```diff
@@ -0,0 +1,74 @@
+description: DC provides up to this standard payment amount under the Temporary Assistance for Needy Families program, based on household size.
+
+metadata:
+  unit: currency-USD
+  period: month
+  breakdown:
+    - range(1, 11)
+  label: DC TANF standard payment amount
+  reference:
+    - title: TANF for District Families
+      href: https://dhs.dc.gov/service/tanf-district-families
+    - title: Code of the District of Columbia § 4-205.52(c)
+      href: https://code.dccouncil.gov/us/dc/council/code/sections/4-205.52#(c)
+    - title: DC TANF State Plan 2024 | Section 4 (4)(a) 
+      href: https://dhs.dc.gov/sites/default/files/dc/sites/dhs/service_content/attachments/DC%20TANF%20State%20Plan_Oct-2023.pdf#page=41
+
+1: 
+  2022-10-01: 437
+  2023-10-01: 450
+  2024-10-01: 490
+2: 
+  2022-10-01: 545 
+  2023-10-01: 560
+  2024-10-01: 612
+3: 
+  2022-10-01: 696 
+  2023-10-01: 712
+  2024-10-01: 781
+4: 
+  2022-10-01: 851
+  2023-10-01: 870
+  2024-10-01: 956
+5: 
+  2022-10-01: 983
+  2023-10-01: 1_002
+  2024-10-01: 1_104 
+6: 
+  2022-10-01: 1_155
+  2023-10-01: 1_178
+  2024-10-01: 1_298 
+7: 
+  2022-10-01: 1_325
+  2023-10-01: 1_352
+  2024-10-01: 1_489 
+8: 
+  2022-10-01: 1_463
+  2023-10-01: 1_494
+  2024-10-01: 1_644
+9: 
+  2022-10-01: 1_612
+  2023-10-01: 1_642
+  2024-10-01: 1_811
+10: 
+  2022-10-01: 1_750
+  2023-10-01: 1_786
+  2024-10-01: 1_967
+11: 
+  2023-10-01: 1_884
+12: 
+  2023-10-01: 2_024
```

Comment:
just for posterity, i noticed that `f(12) - f(11) != f(11) - f(10)`, so this isn't an obvious implicit formula

Reply:
Yeah, tried to find patterns but could consolidate in any way